_tests/browser_tests/spec_suites/shared/shared_the-bureau.js

```diff
@@ -8,12 +10,16 @@ describe( 'Beta The Bureau Page', function() {
     page.get();
   } );
 
-  it( 'should properly load in a browser', function() {
+  it( 'should properly load in a browser',
+  function() {
```

Comment:
Agreed. There's still a few line-length errors where I couldn't figure out a clean way to indent. Do y'all have any suggestions for a case like this?

```
  it( 'should include General Inquiries contact details', function() {
    expect( page.giEmail.getAttribute( 'href' ) ).toBe( 'mailto:info@consumerfinance.gov'
  } );
```